sdk/communication/communication-rooms/test/public/roomsClient.spec.ts

```diff
@@ -128,6 +128,23 @@ describe("RoomsClient", function () {
       await client.updateRoom(roomId, options);
     });
 
+    it("successfully list rooms", async function () {
+      const listRoomsResult = await client.listRooms({});
+
+      assert.isDefined(listRoomsResult);
+      assert.isNotEmpty(listRoomsResult);
```

Comment:
`listRoomsResult` is an async iterator so I don't believe this is valid assertion https://www.chaijs.com/api/assert/#method_isnotempty

Reply:
I believe it would get the count of the enumerable object from the iterator, so it's still valid. However, the collection can be empty since ListRooms does not always return a value. I will remove this assertion.